powerauth-backend-tests/src/test/java/com/wultra/security/powerauth/test/v31/PowerAuthApiTest.java

```diff
@@ -245,6 +243,40 @@ void activationListForUserTest() throws PowerAuthClientException {
         assertNotEquals(0, listResponse.size());
     }
 
+    @Test
+    void testGetActivationListForUserPagination() throws PowerAuthClientException {
+        // Prepare the base GetActivationListForUserRequest
+        GetActivationListForUserRequest request = new GetActivationListForUserRequest();
+        request.setUserId(config.getUserV31());
+        request.setApplicationId(config.getApplicationId());
+
+        // Create multiple activations for the test user
+        for (int i = 0; i < 10; i++) {
+            powerAuthClient.initActivation(request.getUserId(), request.getApplicationId());
+        }
+
+        // Prepare the query parameters for the first page
+        Map<String, String> queryParams1 = new HashMap<>();
```

Comment:
I prefer using the modifier `final` (everywhere) to ensure we are not reassigning the variable. Moreover, here you may use the new API `Map#of()`.